### ISSUE TITLE
Dependency installation error (Python 3.4, Windows)

### ISSUE THREAD

**Issue by dreikanter** [Python-Markdown/markdown#302]
I have two packages dependent on Python Markdown, and recently got identical errors trying to install these packages on Windows, using clean Python 3.4 environment.

```
D:\src\markdown-grid>python setup.py install > install.log
zip_safe flag not set; analyzing archive contents...
error: C:\Users\Alex\AppData\Local\Temp\easy_install-qf3nhntg\Markdown-2.4-py3.4-win-amd64.egg.tmp\markdown\blockparser.pyc: The process cannot access the file because it is being used by another process
```

There is no such error when I'm using Python 3.3. Manual Python Markdown installation (`pip install markdown`) also works fine.

Here is `setup.py` defining dependency for one of my packages:

``` python
#!/usr/bin/env python

import sys
from setuptools import setup


setup(
    name='mdx_grid',
    description='Grid Extension for Python-Markdown',
    version='0.2.1',
    license='MIT',
    author='Alex Musayev',
    author_email='alex.musayev@gmail.com',
    url='http://github.com/dreikanter/markdown-grid',
    long_description=open('README.md').read(),
    package_dir={'mdx_grid': '.'},
    py_modules=['mdx_grid'],
    platforms=['any'],
    install_requires=[
        'markdown',
    ],
)
```

I've already tried to define `dependency_links` in the setup file to install markdown package directly from GitHub, but this doesn't help either.

**Comment by waylan** [comment]
The nature of the error message makes me think it may be an issue on your system. Are you sure there are no other python processes running (the message suggests there is)? If so, have you killed those before trying again?

And after looking at the [release notes](https://docs.python.org/3.4/whatsnew/3.4.html#whatsnew-pep-453) are you sure you are using the Python 3.4 version of pip with Python 3.4?

**Comment by dreikanter** [comment]
There are no other python processes except setup.py running at the same time. And there is only one Python version on the PATH. I double checked my pip version — it's correct.

Also I tryed more dependent packages from [here](https://github.com/waylan/Python-Markdown/wiki/Third-Party-Extensions) and got the identical error. Other packages from `install_requires` installs fine but Python Markdown setup causes a systematic crash for some reason.

```
Searching for markdown
Reading https://pypi.python.org/simple/markdown/
Best match: Markdown 2.4
Downloading https://pypi.python.org/packages/any/M/Markdown/Markdown-2.4.win-amd64.exe#md5=87479a3d05c9c05a1ab5d610ee4f99ec
Processing Markdown-2.4.win-amd64.exe
error: C:\Users\Alex\AppData\Local\Temp\easy_install-e4_b5ut7\Markdown-2.4-py3.4-win-amd64.egg.tmp\markdown\blockparser.pyc: The process cannot access the file because it is being used by another process
```

I found a workaround for the problem. The error reproduces with 64-bit Python version only. When I switched to x86, the setup test passed fine.

So the issue could be closed now. The problem is outside this package sources anyway. Most likely, it caused by some weird Windows behavior or a bug in the specific Python/setuptools version.
